packages/ses/index.d.ts

```diff
@@ -194,6 +194,14 @@ export interface Assert {
   makeAssert: MakeAssert;
 }
 
```

Comment:
As long as this is in a separate commit when it merges, it will have my blessing. I use a workflow like https://abhinavg.net/posts/restacking-branches/. If you prefer to squash this through the Github button, I would prefer this to be a separate PR.

Reply:
My plan was to do an interactive rebase and squash some commits (especially the back-and-forth changes) to leave a cleaner history and then merge, no squash

Reply:
Done, down to 3 meaningful commits